docs/src/environments.md

```diff
@@ -14,7 +14,7 @@ julia> mkdir("MyProject")
 julia> cd("MyProject")
 /Users/kristoffer/MyProject
 
-(v1.0) pkg> activate .
+(v1.0) pkg> activate . # "." means the current directory
```

Comment:
One problem is that now this command does not work to copy-paste any more.

Reply:
My mistake, should have tested in-REPL. How about adding a comment to the previous line like so?
```julia
 julia> cd("MyProject") # we can now use "." instead of a longer relative or full path
```
Might need some wordsmithing, but would be copy-paste friendly.